config/routes.rb

```diff
@@ -106,6 +106,8 @@
   post 'visualizations/shorten_url', to: 'visualizations#shorten_url'
   get 'visualizations.json', to: 'visualizations#index'
 
+  get 'amr_heatmap.json', to: 'amr_heatmap#amr_counts'
```

Comment:
Change `amr_heatmap.json` to `amr_heatmap/amr_counts.json`.

Reply:
Done!